### PR TITLE
Adding migration information to Auth0 doc

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Adding up-to-date images and instructions for initializing the Auth0 migration process

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

**Review by mirnawong1 — APPROVED** [review]
lgtm @matthewshaver !

File: website/docs/docs/cloud/manage-access/auth0-migration.md [hunk 1]
```diff
@@ -16,6 +16,18 @@ If you have not yet configured SSO in dbt Cloud, refer instead to our setup guid
 
 <Snippet src="auth0-uri" />
 
+## Initiate the migration
+
+The Auth0 migration feature is being rolled out incrementally to customers who have SSO features already enabled. When the migration option has been added to your account, you will see **SSO Updates Available** on the right side of the menu bar, near the settings icon.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/manage-access/auth0-migration.md]:
```suggestion
The Auth0 migration feature is being rolled out incrementally to customers who have SSO features already enabled. When the migration option has been enabled on your account, you will see **SSO Updates Available** on the right side of the menu bar, near the settings icon. 
```

File: website/docs/docs/cloud/manage-access/auth0-migration.md [hunk 2]
```diff
@@ -16,6 +16,18 @@ If you have not yet configured SSO in dbt Cloud, refer instead to our setup guid
 
 <Snippet src="auth0-uri" />
 
+## Initiate the migration
```
Comment (matthewshaver) [inline website/docs/docs/cloud/manage-access/auth0-migration.md]:
```suggestion
## Start the migration
```

**Review by mirnawong1 — APPROVED** [review]
great title change! looks fab @matthewshaver !

**Pull request merged**